src/metric.jl

```diff
@@ -179,6 +179,47 @@ function reset!(metric :: MSE)
   metric.n_sample = 0
 end
 
+doc"""
+    NMSE
+
+Normalized Mean Squared Error
+
+```math
+\sum_i (\frac{label_i - pred_i}{label_i})^2
```

Comment:
Can you also add a reference? I am not familiar with NMSE but looking around online I found https://rem.jrc.ec.europa.eu/RemWeb/atmes2/20b.htm which seems to be different from the definition used here.